docker-compose.yml

```diff
@@ -41,6 +44,22 @@ services:
     depends_on:
       - redpanda
 
+  cardemo:
```

Comment:
the current docker-compose file will use Redpanda and related UI/datagen. The cardemo is independent and I think we should create a new docker-compose file in a new folder

Reply:
we can add profile to match different scenario